### PR TITLE
Add few test cases for 100% coverage

### PR DESCRIPTION
**Pull request by aldychris**
Just adding few test cases for better coverage for 2 files

String validator
Before
stringValidator_before
After
stringValidator_after

Pubspec helper
Before
pubspechelper_before
After
pubspechelper_after

## Pre-launch Checklist

- [√ ] I read the [Contribute](https://docs.serverpod.dev/contribute) page and followed the process outlined there for submitting PRs.
- [√] This update contains only one single feature or bug fix and nothing else. (If you are submitting multiple fixes, please make multiple PRs.)
- [√] I read and followed the [Dart Style Guide](https://dart.dev/guides/language/effective-dart/style) and formatted the code with [dart format](https://dart.dev/tools/dart-format).
- [√ ] I listed at least one issue that this PR fixes in the description above.
- [√ ] I updated/added relevant documentation (doc comments with `///`), and made sure that the documentation follows the same style as other Serverpod documentation. I checked spelling and grammar.
- [x] I added new tests to check the change I am making.
- [√ ] All existing and new tests are passing.
- [x ] Any breaking changes are documented below.

### CODE REVIEW

File: tools/serverpod_cli/test/util/string_validator_test.dart [hunk 1]
```diff
@@ -34,4 +34,16 @@ void main() {
...
+    test('isValidTagName returns true for valid tag names', () {
+      expect(StringValidators.isValidTagName('valid-tag-name'), isTrue);
+    });
+    test('returns false for invalid tag names', () {
+      expect(StringValidators.isValidTagName('InvalidTagName'), isFalse);
+      expect(StringValidators.isValidTagName('invalidTagName'), isFalse);
+      expect(StringValidators.isValidTagName('Invalid_Tag_Name'), isFalse);
+      expect(StringValidators.isValidTagName('Invalid-Tag-Name'), isFalse);
+    });
+  });
```
Comment (SandPod) [inline tools/serverpod_cli/test/util/string_validator_test.dart:48]: Suggestion: Make each test cover a single scenario.

When future developers break this, it is going to be more helpful if they can get a specific test specifying what has gone wrong. Please break these up with good descriptions around what scenario we are validating.

**Review by SandPod — CHANGES_REQUESTED** [review]
Awesome, thank you for taking us to 💯%

Only one comment around the test structure and we can merge this.

File: tools/serverpod_cli/test/util/string_validator_test.dart (hunk 1 above)
Comment (aldychris) [reply tools/serverpod_cli/test/util/string_validator_test.dart]: The test cases have been broken down into specific tests

**Review by SandPod — APPROVED** [review]
LGTM! 👍

**Pull request merged**